src/wallet/rpc/spend.cpp

```diff
@@ -787,6 +796,8 @@ RPCHelpMan fundrawtransaction()
                                     },
                                 },
                              },
+                            {"max_tx_weight", RPCArg::Type::NUM, RPCArg::Default{MAX_STANDARD_TX_WEIGHT}, "The maximum acceptable transaction weight.\n"
```

Comment:
Nit: might be good to define this in one place since you use it multiple times.

Reply:
What should be defined?
The help text ? not sure if this is a good idea though so leaving this as is!